dist/L.Control.Locate.css

```diff
@@ -1,5 +1,5 @@
 /*! Version: 0.44.0
```

Comment:
As per https://github.com/domoritz/leaflet-locatecontrol/blob/gh-pages/README.md#developers, please don't include the compiled css.